framework/src/main/java/org/checkerframework/common/wholeprograminference/WholeProgramInferenceScenes.java

```diff
@@ -88,6 +93,25 @@
 
     private final WholeProgramInferenceScenesHelper helper;
 
+    /**
+     * The stub file format requires enums and classes to be output differently. This set keeps
```

Comment:
When writing documentation, the first sentence should be the description of the thing.  Then you can write any additional helpful information.  (So, in this case, the first and second sentence should be switched.) This comment also applies to the Javadoc for `types`.